arch/ext/Sdtrig.yaml

```diff
@@ -23,6 +23,30 @@ description: |
   matches performs the action configured for that trigger.
 
   Triggers do not fire while in Debug Mode.
+params:
```

Comment:
CI is complaining about 2 "params" sections, since there's the original one below. Let's merge these.